phpBB/adm/index.php

```diff
@@ -53,12 +53,19 @@
 $mode			= $request->variable('mode', '');
 
 // Set custom style for admin area
-$template->set_custom_style(array(
-	array(
-		'name' 		=> 'adm',
-		'ext_path' 	=> 'adm/style/',
-	),
-), $phpbb_admin_path . 'style');
+/** @var \phpbb\template\base $template */
+$template->set_custom_style(
```

Comment:
This was changed to support a single form macro file in both adm and prosilver ... so might not be needed